composer.json

```diff
@@ -1,9 +1,9 @@
 {
-    "name": "composer/installers",
-    "type": "composer-installer",
-    "license": "MIT",
+    "name":        "dreamfactory/installers",
```

Comment:
Mind fixing this? Also I think the formatting on the `composer.json` is fine, no need to format it.